### ISSUE TITLE
Merge into drupal.org/project/log

### ISSUE THREAD

**Issue by mstenta** [zaporylie/log#18]
Hi @zaporylie and @eiriksm - I'm the maintainer of the log module on drupal.org: http://drupal.org/project/log

Thanks for putting together this 8.x port of the module! I've been testing and reviewing it for the past few days and it's looking great!

I'd like to figure out the best way to merge this work into the official drupal.org project repo and continue with development. Are you interested in becoming co-maintainers of the module on drupal.org? I think our goals are aligned and I would welcome the help!

I am getting started on the 8.x-2.x version of farmOS (the farm record keeping distribution that was the first use-case for the Log module), so I am eager to finish getting Log 8.x up to parity with the 7.x version. I am also considering using Log in the [Ledger module](http://drupal.org/project/ledger) in Drupal 8. Therefore my goal is to keep Log relatively simple and flexible, so it can be used in lots of different record-keeping use-cases.

**Comment by mstenta** [comment]
We will also need to decide what to do with the existing issues and pull request in this Github repo. Should we try to complete them all before merging into d.o? Or copy them over?

**Comment by mstenta** [comment]
@zaporylie and I chatted earlier today and decided to merge this branch into the official repo at http://drupal.org/project/log.

We will do that once #22 and #23 are closed. Any remaining tasks in the issue queue will be copied to the https://www.drupal.org/project/issues/log.

**Comment by eiriksm** [comment]
Awesome!

Great work @zaporylie and @mstenta !

**Comment by mstenta** [comment]
You too @eiriksm ! Great work! Excited to see Log 8.x-1.x take shape so quickly!

**Comment by mstenta** [comment]
Well, after getting sidetracked for a few months... this is finally done! Voila: https://www.drupal.org/project/log/releases/8.x-1.x-dev

I basically created a new 8.x-1.x branch in the drupal.org repository, with two initial commits that simply include the default output of console generate. Then, I merged this repo's master branch into that, favoring this branch in all conflicts. Adding the two console generate commits provides a bit of comparison if anyone needs it in the future.

I created a new development snapshot release, so now we can start creating 8.x-1.x issues in the drupal.org/project/log issue queue for continued development. I will copy the currently open issues from this repo, and I have a few of my own that I will add as well.

I also have some follow-up commits that clean a few things up... I will organize them and push them up to drupal.org soon...

Thanks again @zaporylie and @eiriksm!!! Your contributions are now included on your drupal.org user profiles:

https://www.drupal.org/u/zaporylie

https://www.drupal.org/u/eiriksm
